app/tests/e2e/test_proof.py

```diff
@@ -0,0 +1,287 @@
+import asyncio
+from typing import Optional, Any, Dict
+import json
+
+from aries_cloudcontroller import (
+    AcaPyClient,
+    IndyRequestedCredsRequestedAttr,
+    IndyRequestedCredsRequestedPred,
+)
+from aries_cloudcontroller.model.indy_pres_spec import IndyPresSpec
+from httpx import AsyncClient
+import pytest
+
+from app.facades.acapy_ledger import create_pub_did
+from app.admin.governance.schemas import SchemaDefinition, create_schema
+from app.dependencies import MEMBER_AGENT_URL
+from app.tests.utils_test import get_random_string
+
+BASE_PATH = "/generic/proof"
+
+
+async def get(response: Optional[Any] = None):
+    if response:
+        return response
+
+
+@pytest.yield_fixture(scope="module")
+def event_loop(request):
+    loop = asyncio.get_event_loop_policy().new_event_loop()
+    yield loop
+    loop.close()
+
+
+@pytest.fixture(scope="module")
+async def schema_definition(yoma_agent_module_scope: AcaPyClient) -> Dict[str, Any]:
+    definition = SchemaDefinition(
+        name="test_schema", version="0.3", attributes=["speed"]
+    )
+
+    public_did = await create_pub_did(yoma_agent_module_scope)
+    print(f"created did: {public_did}")
+
+    schema_definition_result = await create_schema(definition, yoma_agent_module_scope)
+    print(schema_definition_result)
+
+    print(f"created schema {str(schema_definition_result)}")
+    return (schema_definition_result).dict()
+
+
+@pytest.fixture(scope="module")
+async def credential_definition_id(
+    async_client_bob_module_scope: AsyncClient, schema_def: Dict[str, Any]
+) -> str:
+    # when
+    response = await async_client_bob_module_scope.post(
+        MEMBER_AGENT_URL + "/admin/governance/credential-definitions",
+        data=json.dumps(
+            {
+                "support_revocation": False,
+                "schema_id": schema_def["schema_id"],
+                "tag": get_random_string(5),
+            }
+        ),
+    )
+    result = response.json()
+
+    print(f"created definition {str(result)}")
+    return result["credential_definition_id"]
```

Comment:
This is copied exactly from the issuer tests it seems. I think we should extract them into a seperate file to avoid duplication